examples/draw-modify-trace-snap.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: example.html
+title: Draw, modify, trace and snap
+shortdesc: Using the draw, modify, trace and snap interactions to edit geometries in the context of existing data.
```

Comment:
Minor (existing text makes it sound like there is a trace interaction):
```suggestion
shortdesc: Using the draw, modify, and snap interactions to edit existing geometries with tracing enabled.
```